stable/redis/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 10.2.1
+version: 10.2.2
```

Comment:
I would suggest you bump the minor chart version for this change

Reply:
done